EnglishFilter/sections/foreign.txt

```diff
@@ -1086,6 +1086,8 @@ m.youku.com##body > [style="width: 100px; height: 100px; position: fixed; top: a
 m.dnvod.tv#%#AG_onLoad(function() { setTimeout(function() { document.getElementById("player").removeAttribute("hidden"); }, 3000); });
 m.dnvod.tv#$#.top-bar > h3.text-over { display:none!important; }
 m.dnvod.tv#$#.video-box > .video-box { display:none!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/27253
```

Comment:
Move it to the top of #Chinese section. So it is the next element below #Chinese, so it will look like this:

!---------------------------------------------------------------------
!------------------------------ Chinese ------------------------------
!---------------------------------------------------------------------
! https://github.com/AdguardTeam/AdguardFilters/issues/27253
ruanyifeng.com###alpha-inner > div > p:has(> a[href^="http://qr.jirengu.com"])
! https://github.com/AdguardTeam/AdguardFilters/issues/35258
||static.tohomh123.com/js/funasd.js